redbeams/src/main/java/com/sequenceiq/redbeams/domain/stack/DBStack.java

```diff
@@ -194,4 +193,29 @@ public Crn getOwnerCrn() {
     public void setOwnerCrn(Crn ownerCrn) {
         this.ownerCrn = ownerCrn;
     }
+
+    public String getAccountId() {
+        return ownerCrn != null ? ownerCrn.getAccountId() : null;
+    }
+
+    // careful with toString - it may cause database accesses for nested entities
```

Comment:
Do we need this comment anymore?

Reply:
It's actually new. I felt I should include it because of a warning I read about including toString() on JPA entities - when an entity has lazy-loaded fields, a call to toString() can cause database access when you aren't necessarily expecting it.